static/app/components/events/interfaces/sourceMapsDebuggerModal.tsx

```diff
@@ -673,15 +676,24 @@ function HasDebugIdChecklistItem({
           </p>
           {defined(sourceMapsDocLinks.sentryCli) && (
```

Comment:
From
 
```
If you are using a [bundlerPluginRepoLink:Sentry Plugin for your Bundler], the plugin needs to be active when building your production app. You cannot do two separate builds, for example, one for uploading to Sentry with the plugin being active and one for deploying without the plugin. The plugin needs to be active for every build.
```

To `bundlerPluginRepoLink` = https://docs.sentry.io/platforms/react-native/manual-setup/metro/

```
The [bundlerPluginRepoLink:Sentry Metro Plugin] needs to be active when building your production app. You cannot do two separate builds, for example, one for uploading to Sentry with the plugin being active and one for deploying without the plugin. The plugin needs to be active for every build.
```